Cargo.toml

```diff
@@ -29,8 +29,8 @@ blitzar = { version = "4.0.0" }
 bumpalo = { version = "3.11.0" }
 bytemuck = {version = "1.16.3", features = ["derive"]}
 byte-slice-cast = { version = "1.2.1", default-features = false }
-clap = { version = "4.5.4" }
-criterion = { version = "0.5.1" }
+clap = { version = "=4.4.9" }
+criterion = { git = "https://github.com/bheisler/criterion.rs.git", rev = "260e2f1c788227096cf483c0c2a1933968bbe27b" }
```

Comment:
This is in order to make sure we don't get `itertools` version discrepancy. See https://github.com/bheisler/criterion.rs/issues/836